logos/pandascrow/pandascrow.svg

```diff
@@ -0,0 +1,8 @@
+<path fill-rule="evenodd" clip-rule="evenodd" d="M0 8.5C0 11.846 1.93331 14.7404 4.7438 16.1271C2.98661 19.7103 2 23.7399 2 28C2 42.2226 12.9969 53.8768 26.9518 54.9235C26.9496 54.9158 26.949 54.908 26.95 54.9C26.9867 54.6067 26.9696 48.8285 26.9578 45.7673C22.9711 44.8424 20 41.2681 20 37C20 32.0294 24.0294 28 29 28C33.9706 28 38 32.0294 38 37C38 41.2982 34.987 44.8926 30.9579 45.7864C30.9696 48.8525 30.9866 54.6073 30.95 54.9C30.9487 54.9108 30.9444 54.9213 30.9375 54.9316C44.9449 53.9384 56 42.2602 56 28C56 23.5656 54.931 19.3808 53.0366 15.6894C55.4184 14.1834 57 11.5264 57 8.5C57 3.80558 53.1944 0 48.5 0C45.4736 0 42.8166 1.58163 41.3106 3.96341C37.6192 2.06902 33.4344 1 29 1C24.2587 1 19.8028 2.22212 15.9302 4.36853C14.4781 1.76265 11.6949 0 8.5 0C3.80558 0 0 3.80558 0 8.5Z" fill="#093373"/>
```

Comment:
Nice @prezine
Looks like there's some more room for optimisation here.

Ensure your SVG & PNG logos have been properly optimised with tools like [SVGOMG](https://jakearchibald.github.io/svgomg/) & [Squoosh](https://squoosh.app/) (OxiPNG works well) respectively